spec/unit/gis_robot_suite/structural_metadata_builder_spec.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe GisRobotSuite::StructuralMetadataBuilder do
+  describe '.build' do
+    subject(:result) { described_class.build(cocina_model, druid, objects) }
```

Comment:
Why does the cocina_model that you're invoking this with already have structural?

Reply:
not sure, will give the tests another read and try to figure out if there's room for improvement here